impl/src/main/java/org/jboss/cdi/tck/interceptors/tests/contract/aroundConstruct/bindings/AroundConstructTest.java

```diff
@@ -46,6 +47,8 @@
  * @author Matus Abaffy
  */
 @SpecVersion(spec = "interceptors", version = "1.2")
+// Use of interceptors in beans.xml not supported in Lite
+@Test(groups = CDI_FULL)
```

Comment:
Move to `full` package is appropriate then?

(Frankly, the more I think about it, the more I find the entire `full` package idea totally worthless.)

Reply:
This comment also applies to several other classes in this PR.

Reply:
+1, should go into `full` package
I don't think its worthless if we manage to really move all the affected tests there as it will make it clearer which things are tested in which env.